managers/signals.py

```diff
@@ -0,0 +1,65 @@
+import logging
+from typing import Any
+from typing import Set
+
+from django.db.models.signals import m2m_changed
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from employees.models import TaskActivityType
+from managers.models import Project
+from users.models import CustomUser
+
+logger = logging.getLogger(__name__)
+
+
```

Comment:
shouldn't we prevent signals from being duplicated? https://docs.djangoproject.com/en/2.2/topics/signals/#preventing-duplicate-signals
@rwrzesien, what would be your suggestions here?

Reply:
I will wait for @rwrzesien opinion, because I am not sure what I should have to do now.

Reply:
I think we won't have this problem here because we don't connect it manually, but is nice that you have thought about this case.